Release r1.2 (Fall'25 M4) (#34)
camaraproject/PredictiveConnectivityData: CHANGELOG.md

* for subsequent release-candidate(s), only the delta to the previous release-candidate
* for a public release, the consolidated changes since the previous public release

# r1.2

## Release Notes

> This release contains the definition and documentation of 

Reviewer: ```suggestion
This public release contains the definition and documentation of 
```

* predictive-connectivity-data v0.1.0

The API definition(s) are based on
* Commonalities v0.6.0
* Identity and Consent Management v0.4.0

## predictive-connectivity-data v0.1.0

> This is a release candidate for the CAMARA Meta Release Fall25 release of the Predictive Connectivity Data API, version v0.1.0. It contains mainly alignments with the Commonalities v0.6.0.

Reviewer: ```suggestion
This is the public release for the CAMARA Meta Release Fall25 release of the Predictive Connectivity Data API, version v0.1.0. It contains mainly alignments with the Commonalities v0.6.0.
```

- API definition **with inline documentation**:
  - OpenAPI [YAML spec file](https://github.com/camaraproject/PredictiveConnectivityData/blob/r1.2/code/API_definitions/predictive-connectivity-data.yaml)
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PredictiveConnectivityData/r1.2/code/API_definitions/predictive-connectivity-data.yaml&nocors)
  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/PredictiveConnectivityData/r1.2/code/API_definitions/predictive-connectivity-data.yaml)

In the following there is the list of the modifications with respect to the previous release.

### Added
* N/A

### Changed
* Alignment with Commonalities r3.3 by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/33

### Fixed
* N/A

### Removed
* N/A

 ### New Contributors
* @eric-murray made their first contribution in https://github.com/camaraproject/PredictiveConnectivityData/pull/30

> **Full Changelog**: https://github.com/camaraproject/PredictiveConnectivityData/compare/r1.1...r1.2

Reviewer: For the first public release, usually all commits that are part of that release are listed
```suggestion
**Full Changelog**: https://github.com/camaraproject/PredictiveConnectivityData/commits/r1.2
```

  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/PredictiveConnectivityData/r1.2/code/API_definitions/predictive-connectivity-data.yaml)

In the following there is the list of the modifications with respect to the previous release.

### Added
* Add precision, height and signal strenght to the wip version by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/25

### Changed
* Alignment with Commonalities r3.3 by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/33
* Define first WIP version of Predictive Connectivity Data API by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/20
* Align commonalities with v0.6-rc1 by @albertoramosmonagas in https://github.com/camaraproject/PredictiveConnectivityData/pull/21

### Fixed
* N/A

### Removed
* N/A

 ### New Contributors
* @eric-murray made their first contribution in https://github.com/camaraproject/PredictiveConnectivityData/pull/30

> * @hdamker made their first contribution in https://github.com/camaraproject/PredictiveConnectivityData/pull/3

Reviewer 2: I suggest to omit this line as it was an administrative PR.

```suggestion
```
